Tests/SpeziAccountMacrosTests/Utils/MacroAssertions.swift

```diff
@@ -0,0 +1,44 @@
+//
+// This source file is part of the Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftSyntax
+import SwiftSyntaxMacroExpansion
+import SwiftSyntaxMacros
+import SwiftSyntaxMacrosGenericTestSupport
+import Testing
+
+// Adapted function from https://github.com/swiftlang/swift-syntax/blob/main/Sources/SwiftSyntaxMacrosTestSupport/Assertions.swift
+func assertMacroExpansionWithSwiftTesting(
```

Comment:
A more meta level comment for @PSchmiedmayer. We currently maintain [XCTestExtensions](https://github.com/StanfordBDHG/XCTestExtensions) which by its name is pretty XCTest specific.

This method might be a good fit to eventually move out to some `SwiftTestingExtensions` package (this use case is hopefully natively supported by SwiftSyntax at some point). This should not block this PR and it's fine to keep it here for now. I just want to start discussing how we would want to proceed with things like this?

Reply:
Agree, I think we should create a small package for that. Agree that XCTestExtensions is not the right place for that. Created a repo for this: https://github.com/StanfordBDHG/SwiftTestingExtensions